source/layout.erb

```diff
@@ -122,6 +122,27 @@
         ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
         var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
       })();
+
+     $('#edit-page').on('click', function(event) {
+         event.preventDefault();
+         var editPageUrl = event.target.getAttribute('href')
```

Comment:
End statements with semicolons per the [Ember Style Guide](https://github.com/emberjs/ember.js/blob/master/STYLEGUIDE.md#semicolons).